server/routes/testutils/appSetup.ts

```diff
@@ -45,7 +44,7 @@ function appSetup({
 
   app.set('view engine', 'njk')
 
-  nunjucksSetup(app, path)
+  nunjucksSetup(app)
 
```

Comment:
I'm trying to think of any consequence of this - like will it alter the path for running in any of the modes (test, integration test, docker)? I think its neater and less complicated, and if this works in all the modes then I'm not sure why the path module was originally there!

Reply:
I cant immediately see why it would break in any case.. Integration tests passed in the pipeline. I guess we should just check that it works in Dev